test/shopify-cli/identity_auth/env_auth_token_test.rb

```diff
@@ -0,0 +1,73 @@
+require "test_helper"
+
+module ShopifyCLI
+  class IdentityAuth
+    class EnvAuthTokenTest < MiniTest::Test
+      def test_partners_token_present
+        # Given
+        Environment.expects(:auth_token).returns("token")
```

Comment:
The Given part should not have expectations, but stubs.

```suggestion
        Environment.stubs(:auth_token).returns("token")
```